/slam_node:
  ros__parameters:
    velocity_estimation_frequency: 40

    # This sets the mode of operation. Set true for slam and false for localization
    mapping_mode: true
    track_map: ""

    # Defines the starting position of the car
    starting_position: [ -2.0, 0.0, 0.0 ] #before -7.5
    starting_position_covariance: [ 0.5, 0.1, 0.1 ]

    # The max distance perception measurements are accepted
    perception_range: 13.0

    # After how many time steps the optimization algorithm would be executed
    optimization_interval: 4

    # The max distance at which two cones would be associated
    association_threshold: 1.9
    min_observations: 4

    # iSAM parameters
    relinearize_threshold: 0.1
    relinearize_skip: 1

    # Big orange cones for starting/finish line
    left_orange: [0.0, -2.5]
    right_orange: [0.0, 2.5]

    lap_counter_cooldown: 10

    logger: true